### DISCUSSION TITLE
C# → someone has achieved certain Godot Editor integration of VSCode

### DISCUSSION THREAD

**Discussion by azur-wolve** [godotengine/godot-proposals#5178] [Editor]
https://www.reddit.com/r/godot/comments/wl3u86/embedding_vscode_directly_into_the_godot_editor/
Maybe could be useful for future C# integration with the editor.

**Comment by anderlli0053** [comment]
Well, i'd prefer the VSCodium over the original VSCode, because the devs removed the tracking from the VSCode and made its own unique distribution based off VSCode and named it VSCodium --> https://github.com/VSCodium/vscodium

**Reply by azur-wolve** [reply]
sounds good
But it has the tools and plugins that VSCode has?

**Reply by anderlli0053** [reply]
> sounds good But it has the tools and plugins that VSCode has?

Yes, you can install the same extensions and all, because it is practically the fork of the VSCode, just with removed tracking and all that.

The best thing for you is to try it out yourself and see. 

If you use Scoop it's also available in many buckets, including mine. -> https://scoop.sh/

**Reply by RedMser** [reply]
It should theoretically work with any external code editor. The addon is currently just checking for hard coded window title.

**Reply by SilencedPerson** [reply]
GodotOS soon.
